tests/unit/Statement/Grouper/NullStatementGrouperTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Tests\Statement\Grouper;
+
+use PHPUnit_Framework_TestCase;
+use Wikibase\DataModel\Services\Statement\Grouper\NullStatementGrouper;
+use Wikibase\DataModel\Snak\PropertyNoValueSnak;
+use Wikibase\DataModel\Statement\StatementList;
+
+/**
+ * @covers Wikibase\DataModel\Services\Statement\Grouper\NullStatementGrouper
+ *
+ * @licence GNU GPL v2+
+ * @author Thiemo Mättig
+ */
+class NullStatementGrouperTest extends PHPUnit_Framework_TestCase {
+
+	public function test() {
```

Comment:
I thought PHPUnit would get mad if you just called a method test. Perhaps not. Not sure now it will actually run it. Better to give the test a name :)